index.html

```diff
@@ -4,20 +4,48 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="permaLogin.css">
+    <title>OpenBakalari login</title>
+    <style id="dyn"></style>
   </head>
-  <body>
     <script src="renderer.js"></script>
-    <header>
-      <h1>OpenBakaláři</h1>
-    </header>
+    <center id="errorstatus"></center>
     <main>
-      <form onsubmit="login()">
-        <input placeholder="Doména" type="url" name="domain" id="inst"><br>
-        <input placeholder="Uživatelské jméno" type="text" name="user" id="user"><br>
-        <input placeholder="Heslo" type="password" name="pass" id="pass"><br>
-        <input type="submit" value="Přihlásit se">
-      </form>
+      <header>
+        <center><h1>OpenBakaláři</h1></center>
+      </header>
+      <article>
+          <form onsubmit="login()">
+          <span class="hints">Domena: </span><br>
+          <input placeholder="Doména" type="url" name="domain" id="inst"><br>
+          <span class="hints">Uzivatelske jmeno: </span><br>
+          <input placeholder="Uživatelské jméno" type="text" name="user" id="user"><br>
+          <span class="hints">Heslo: </span><br>
+          <input placeholder="Heslo" type="password" name="pass" id="pass"><br>
+          <input type="submit" value="Přihlásit se">
+        </form>
+      </article>
+      <hr>
+      <span class="other">Custom wrapper pro bakalare</span>
     </main>
+    <script src="themeParser.js"></script>
+    <script>
+      onload=theme(`{"colors":{
+    "back":"#303446",
+    "inputs":"#a5adce",
+    "greytext":"#000000",
+    "button":"#292c3c",
+    "buttontext":"#b5bfe2",
+    "buttonhov":"#414559",
+    "borderc":"#babbf1",
+    "text1":"#a6d189", 
+    "text2":"#e5c890",
+    "text3":"#0000000",
+    "text4":"#c6d0f5"}, 
+"other":{
+    "round":10, 
+    "border":1,
+    "font":"monospace"}}`)
+    </script>
```

Comment:
Bylo by lepší vkládat uživatelem vytvořené CSS soubory, taky by to umožňovalo větší customizabilitu.

Reply:
1. Jo, sorry, body jsem omylem oddelal
2. To je schvalne protoze kdyz neni obsah v nejakem wrapperu tak se blbe pozicuje
3.  Placeholder je OK dokud clovek nezacne psat a pokud je jak ja nebo ty, tak zapomene co je vlasne co. Je to proste standart to takhle delat
4. Custom CSS je podle me overkomplikovani. Vem si, ze vetsina lidi, co budou tuhle aplikaci pouzivat jsou gymnazisti nebo studenti cestiny, a ti proste neumi CSS, kdezto pro JSON je jednodusi delat theme engine

Reply:
2. Centeruješ to h1, ne header, takže by to neměl být problém
3. OK
4. Stejně jestli sou to opravdu gymnázisti nebo studenti češtiny, tak nebudou mít čas ani si vytvářet vlastní theme, takže si je buďto stáhnou z internetu, nebo nebudou themovat vůbec.

Reply:
2. ja ale potrebuju wrapper, kterej budu centrovat. Proto jsem to dal do header, login do article atd.
4. Jo, muzes tam pridat moznsot davat vlastni CSS pak, ale na to se blbe dela GUI tool. Na JSON se da jednoduse udelat GUI nebo i CLI tool na designovani.

Reply:
1. A stejně si to pořád neopravil.
2. Proč se vlastně bavíme o vycentrování, když já to ani centrovat nechci.
4. 
> GUI tool
To je celej problém téhle konverzace.

Reply:
ty hade.. ok